Sources/SwiftCliCore/ChartConstructor.swift

```diff
@@ -11,20 +11,34 @@ public class ChartConstructor {
         // NEW LOGIC
         for row in 0...lastrow {
             let row_length = stitch_array[row].count
+            
             if (row == 0){
-                print("first")
-                finished_chart += self.make_top_row(width: row_length)
-                finished_chart += self.make_stitch_row(row: stitch_array[row])
+                finished_chart =  self.make_bottom_row(width: row_length) + finished_chart
+                finished_chart = self.make_stitch_row(row: stitch_array[row]) + finished_chart
```

Comment:
Do you need the `self` here to refer to other methods in the same class? I was under the impression Swift didn't require it.

Reply:
You're right!  I took that out.